Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ however, it has to be formatted properly to pass verification tests.
 
 ### Changed
 - Improved performance of HID descriptor parsing by moving json parsing to a simple custom predicitve parser instead of relying on Unity's json parsing. This should improve domain reload times when there are many HID devices connected to a machine.
+- Disable shortcut support by default. This restores the behaviour from v1.3 and fixes the case of actions not triggering if multiple actions are enabled using the same controls ([case ISXB-254](https://issuetracker.unity3d.com/product/unity/issues/guid/ISXB-254))).
```

Comment:
Do you think we should list this as a change? Or should we call it out as this having side-effects on behavior in prior 1.4.x versions and call out out as a regression? E.g. "Reverted shortcut support being enabled by default as introduced in v.1.4.0. This restores the behavior...."

Reply:
(I think either is fine and its a minor thing in practise if called out as a bug or change, but it might be highlighted when reviewed by RM as a different thing)

Reply:
I think that's fair enough. I've updated it.
I also realised it was in the wrong place as well (wrong version)